sonar-plugin-server/src/main/java/jetbrains/buildserver/sonarplugin/manager/BaseSQSInfo.java

```diff
@@ -15,12 +15,12 @@ public class BaseSQSInfo extends BasePropertiesBean implements SQSInfo {
     public static final String ID = "id";
     public static final String JDBC_URL = "jdbcUrl";
     public static final String JDBC_USERNAME = "jdbcUsername";
-    public static final String JDBC_PASSWORD = "jdbcPassword";
+    public static final String JDBC_PASSWORD = "secure:jdbcPassword";
```

Comment:
What about JSP, shouldn't we change the parameter names in JSP as well?

Reply:
As I understand, it uses different parameters in JSP, later they are converted to parameters of project feature. So no change in needed.